jupyterlab_gcpscheduler/src/data.ts

```diff
@@ -77,68 +82,104 @@ export function removeFromList<T>(list: T[], value: T) {
   }
 }
 
-export const CUSTOM_CONTAINER: Option = {
+export const CUSTOM_CONTAINER: Framework = {
   value: 'customContainer',
   text: 'Custom Container',
+  searchKeywords: [] as string[],
```

Comment:
Do you need the `[] as string[]` here? I would have though the explicit interface definition would suffice.

Reply:
Done. It must have been an IDE delay, I had thought it threw a lint error.